.circleci/config.yml

```diff
@@ -253,21 +256,21 @@ workflows:
       jobs:
          - macos_setup
          - linux_setup
-         - macos_vcs_py2:
-              requires:
-                 - macos_setup
+         #- macos_vcs_py2:
```

Comment:
Usually I remove commented out code, but did you leave these here in case we change our mind later and want to add python2 back in?

Reply:
Yes, I am leaving it there for now.
I will run through all cdat packages again to add py38 at some point. I can clean up that time. Thanks @downiec